src/dataframe/dataframe.jl

```diff
@@ -34,6 +35,8 @@ DataFrame(ds::AbstractDict)
 * `categorical` : `Vector{Bool}` indicating which columns should be converted to
                   `CategoricalVector`
 * `ds` : `AbstractDict` of columns
+* `table`: a "catchall" constructor that accepts any type that implements the
```

Comment:
I wouldn't say "a "catchall" constructor" since you describe an argument rather than a function.

Reply:
Fixed.